Prepare for release 23.6.0 (#408)
PyCQA/flake8-pyi: CHANGELOG.md

  sys.version_info >= (3, 10)` check). This bug has now been fixed.

Other changes:

> * flake8-pyi no longer supports being run on Python 3.7, which is now end-of-life.

Reviewer: Not a native speaker, so please ignore if worse:

```suggestion
* flake8-pyi no longer supports being run on Python 3.7, which has reached its end-of-life.
```
